scripts/sdk-create-pr.sh

```diff
@@ -91,10 +91,13 @@ for service_path in ${work_dir}/sdk_to_push/services/*; do
             git add go.work
         fi
         
-        git commit -m "Generate $service"
-        git push origin "$branch"
         if [[ "$branch" != "main" ]]; then
-            gh pr create --title "Generator: Update SDK /services/$service" --body "$PR_BODY" --head "$branch" --base "main"
+            git commit -m "Generate $service"
```

Comment:
Why do the commit messages need to be different?

Reply:
For the internal SDK there is no PR, so it's useful to include the build ID on the commit message